src/D2L.Bmx/Program.cs

```diff
@@ -21,8 +23,28 @@
 // allow no-sandbox argument for chromium to for passwordless auth with elevated permissions
 var bypassBrowserSecurityOption = new Option<bool>(
 	name: "--experimental-bypass-browser-security",
+	getDefaultValue: () => false,
 	description: ParameterDescriptions.ExperimentalBypassBrowserSecurity );
 
+bypassBrowserSecurityOption.AddValidator( result => {
+	bool bypass = result.GetValueForOption( bypassBrowserSecurityOption );
+	bool isElevated = false;
+	if( OperatingSystem.IsWindows() ) {
+		isElevated = new WindowsPrincipal( WindowsIdentity.GetCurrent() ).IsInRole( WindowsBuiltInRole.Administrator );
+	} else if( OperatingSystem.IsLinux() || OperatingSystem.IsMacOS() ) {
+		isElevated = NativeMethods.GetPosixEuid() == 0;
+	}
+	if( isElevated && !bypass ) {
+		result.ErrorMessage = """
+			Running BMX with elevated permissions is only allowed with the '--experimental-bypass-browser-security' flag.
+			Only include this if you aren't concerned with the security of the host for your Okta organization.
+			""";
+	} else if( !isElevated && bypass ) {
+		result.ErrorMessage
+			= "BMX is not running with elevated permissions, so --experimental-bypass-browser-security must be set to false";
```

Comment:
Isn't it better to say, something along the lines of: don't provide ``-experimental-bypass-browser-security``

Reply:
I'd say this is more than a nit

Reply:
Sure I'll reword